### PR TITLE
[SUPPORTESC-153] Update the integration-checks-reference.md for check `D006`

### PR DESCRIPTION
**Pull request by iscorer**
- Makes it clearer that the URL needs to be specified in Perform List
- Uses similar terminology as check `T006 `
- HelpScout: https://secure.helpscout.net/conversation/1474953799/2345789/

### CODE REVIEW

File: docs/_docs/integration-checks-reference.md
Comment (armchairlinguist) [inline docs/_docs/integration-checks-reference.md:182]: Hey @iscorer ! We just went through some discussion about this text in a couple of earlier PRs (https://github.com/zapier/visual-builder/pull/185, https://github.com/zapier/zapier/pull/48065, https://github.com/zapier/visual-builder/pull/196. Could I bother you to revise this a bit to keep the majority of the existing text? I'd suggest the following, but I'd love to see thoughts from @eliangcs as well.

> When users are setting up a hook-based (aka instant) Trigger, it's important to have
a polling fallback. For example, imagine a Zap that triggers on a new Slack message.
If testing relies on the sending of a webhook, the test won't complete without the user 
sending an actual message in a Slack channel, which is disruptive. 

> Instead, during testing, the Perform List (`performList`) operation fetches a 
(real) recent message using the provided URL for polling and uses it as the test result. 
The polling URL is only used for tests.

Comment (eliangcs) [reply docs/_docs/integration-checks-reference.md:182]: I like @armchairlinguist's revision, which I think has more information from an end user's perspective. 

Also, remember to copy the update to [`z/z` repo](https://github.com/zapier/zapier/blob/695504d2447e04ea88130b3654b0481e405b0157/web/backend/zapier/app_platform/developer_cli/app_checker/checks/app_definition.py#L507-L512).

Comment (iscorer) [reply docs/_docs/integration-checks-reference.md]: Committed the suggested copy above. Re-requesting review. Will follow-up with changes to `z/z` once this PR has been merged.

File: docs/_docs/integration-checks-reference.md
Comment (eliangcs) [inline docs/_docs/integration-checks-reference.md:187]:
```suggestion
Instead, during testing, the Perform List (`performList`) operation fetches a
```

**Review by eliangcs — APPROVED** [review]
Thanks for the change! I have a suggested change on the text formatting. Let me know if you need re-approval.

**Review by armchairlinguist — APPROVED** [review]
This looks great!

**Comment by armchairlinguist** [comment]
You just need a JIRA somewhere to attach this to. Do you have access to the SUPPORTESC project?

**Pull request merged**
